copulas/univariate/gaussian_kde.py

```diff
@@ -180,6 +181,45 @@ def percent_point(self, U):
 
         return X
 
+    def percent_point_fast(self, U):
+        """Compute the inverse cumulative distribution value for each point in U.
+
+        Arguments:
+            U (numpy.ndarray):
+                Values for which the cumulative distribution will be computed.
+                It must have shape (n, 1) and values must be in [0,1].
+
+        Returns:
+            numpy.ndarray:
+                Inverse cumulative distribution values for points in U.
+
+        Raises:
+            NotFittedError:
+                if the model is not fitted.
+        """
+        self.check_fit()
+
+        assert len(U.shape) == 1
```

Comment:
Let's not use a plain `assert` here, which is hard to interpret by the users, but rather an `if` and a `raise ValueError` indicating that we only support 1d arrays.